server_admin/topics/account.adoc

```diff
@@ -2,12 +2,12 @@
 
 == Account Console
```

Comment:
This file was already merged as part of a different PR.

Reply:
No problem, I have restored it back.